kickFlip-project/src/components/form/changePasswordForm/changePasswordForm.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import { ErrorMessage, SignUpDataForm } from '@/types/types';
+import { useDispatch } from '@/services/store';
+import { signUpUser } from '@/services/userSlice';
+import FormField from '@/components/formFields/formField';
+import { responsesErrorsHandler } from '@/utils/utils';
+
+import '../form.css';
+
+export default function ChangePasswordForm() {
+    const [registrationError, setRegistrationError] = useState('');
+
+    const {
+        register,
+        handleSubmit,
+        reset,
+        formState: { errors, isValid },
+    } = useForm<SignUpDataForm>({ mode: 'all' });
```

Comment:
интерфейс SignUpDataForm будет избыточным для этой формы, можно попробовать что-то попроще
```suggestion
  interface ChangePasswordData {
    password: string;
}
   const {
        register,
        handleSubmit,
        reset,
        watch,
        formState: { errors, isValid },
    } = useForm<PasswordForm>({ mode: 'all' });
```

Reply:
Исправил